### DISCUSSION TITLE
How do I add a screenshot to my user feedback?

### DISCUSSION THREAD

**Discussion by rakuyoMo** [getsentry/sentry-cocoa#2961] [Q&A]
I found that `attachScreenshot` and `attachViewHierarchy` are not available in the user feedback.

`SentryScreenshot` and `SentryViewHierarchy` are also not public.

So is there any other way I can achieve this? I haven't found any other interface and it seems that I have to implement the relevant functionality manually myself for now.

I am using iOS and Android SDK

**Comment by brustolin** [comment]
Hello @rakuyoMo.
Right now is not possible to do this automatically.

The only solution I can think of, is for you to take the screenshot and set it in the scope before capturing user feedback.

If you think this would be a interesting feature, I kindly ask you to open an issue.

**Reply by rakuyoMo** [reply]
I now do this by `capture(error:)`  instead of `capture(message:)`.

But I still want to be able to report user feedback via capture Message.

I will create a new issue.

**Reply by armcknight** [reply]
I will close this discussion for now as it hasn't been updated in a while, but feel free to reopen for further discussion or to open an issue to request this feature in the future if you find you'd still like it! Thanks 🙏🏻

**Reply by rakuyoMo** [reply]
I'll follow up in #3007.
